### PR TITLE
Swift examples for location and gimbal docs

### PR DESCRIPTION
**Pull request by Bucimis**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/advanced_use_cases/beacon_integration.md [hunk 1]
```diff
@@ -14,9 +14,23 @@ Once you have your Gimbal Beacons set up and integrated into your app, you can l
...
 ```objc
 [[Appboy sharedInstance] logCustomEvent:@"Entered %@", visit.place.name];
 [[Appboy sharedInstance] flushDataAndProcessRequestQueue];
 ```
 
+{% endtab %}
+{% tab swift %}
+
+```swift
+Appboy.sharedInstance()!.logCustomEvent("Entered %@", visit.place.name)
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/advanced_use_cases/beacon_integration.md]: I think these are supposed to say `Appboy.sharedInstance()?`
e.g. https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/in-app_messaging/customization/#setting-orientation-for-all-in-app-messages

File: _docs/_developer_guide/platform_integration_guides/ios/analytics/location_tracking.md [hunk 2]
```diff
@@ -143,6 +146,20 @@ The following two methods can be used to set the last known location for the use
 
 ```
 
+{% endtab %}
+{% tab swift %}
+
+```swift
+Appboy.sharedInstance()!.user.setLastKnownLocationWithLatitude(latitude: Double, longitude: Double, horizontalAccuracy: Double)
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/analytics/location_tracking.md]: Here too with `sharedInstance()?`*

File: _docs/_developer_guide/platform_integration_guides/ios/analytics/location_tracking.md [hunk 3]
```diff
@@ -143,6 +146,20 @@ The following two methods can be used to set the last known location for the use
...
+```
+
+```swift
+Appboy.sharedInstance()!.user.setLastKnownLocationWithLatitude(latitude: Double, longitude: Double, horizontalAccuracy: Double, altitude: Double, verticalAccuracy: Double)
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/analytics/location_tracking.md]: I think we should replace the `Double` that appears with the actual variableName for this place and the one above

```
Appboy.sharedInstance()!.user.setLastKnownLocationWithLatitude(latitude: latitude, longitude: longitude, horizontalAccuracy: horizontalAccuracy, altitude: altitude, verticalAccuracy: verticalAccuracy)
```

Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/analytics/location_tracking.md]: done

File: _docs/_developer_guide/platform_integration_guides/ios/advanced_use_cases/beacon_integration.md (hunk 1 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/advanced_use_cases/beacon_integration.md]: done

**Comment by Bucimis** [comment]
@EmilyNecciai please merge

**Pull request merged**